!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  arm_led_btn: right_axes_button
  enable_disable_semi_btn: start_button
  gripper_axes: arrow_x
  gripper_max_speed: 2500
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      arm_led_btn: right_axes_button
      enable_disable_semi_btn: start_button
      gripper_axes: arrow_x
      gripper_max_speed: 2500
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      hazmat_enable_btn: left_axes_button
      id: 0
      joint1_dec_axes: rt_axes
      joint1_inc_axes: lt_axes
      joint1_max_speed: 6500
      joint2_axes: right_y_axes
      joint2_max_speed: 9500
      joint3_axes: left_y_axes
      joint3_max_speed: 7000
      joint4_dec_btn: a_button
      joint4_inc_btn: y_button
      joint4_max_speed: 1500
      joint5_dec_btn: lb_button
      joint5_inc_btn: rb_button
      joint5_max_speed: 2500
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      prismatic_axes: arrow_y
      prismatic_max_speed: 5200
      reset_motors_btn: home_button
      semi_x_axes: right_y_axes
      semi_y_dec_axes: rt_axes
      semi_y_inc_axes: lt_axes
      semi_z_axes: left_y_axes
      sensor_box_dec_btn: x_button
      sensor_box_inc_btn: b_button
      sensor_box_max_speed: 2500
      state: true
      type: ''
    state: []
  hazmat_enable_btn: left_axes_button
  joint1_dec_axes: rt_axes
  joint1_inc_axes: lt_axes
  joint1_max_speed: 6500
  joint2_axes: right_y_axes
  joint2_max_speed: 9500
  joint3_axes: left_y_axes
  joint3_max_speed: 7000
  joint4_dec_btn: a_button
  joint4_inc_btn: y_button
  joint4_max_speed: 1500
  joint5_dec_btn: lb_button
  joint5_inc_btn: rb_button
  joint5_max_speed: 2500
  prismatic_axes: arrow_y
  prismatic_max_speed: 5200
  reset_motors_btn: home_button
  semi_x_axes: right_y_axes
  semi_y_dec_axes: rt_axes
  semi_y_inc_axes: lt_axes
  semi_z_axes: left_y_axes
  sensor_box_dec_btn: x_button
  sensor_box_inc_btn: b_button
  sensor_box_max_speed: 2500
state: []
